fastparquet/writer.py

```diff
@@ -1156,6 +1156,8 @@ def write(filename, data, row_group_offsets=None,
                open_with=open_with, mkdirs=mkdirs, remove_with=None,
                stats=stats)
         return
+    if isinstance(partition_on, str):
```

Comment:
The docstring still says that it must be a list of string

Reply:
Ok, with last commit, docstring is updated.